commercial/app/views/masterClasses/masterClasses.scala.html

```diff
@@ -22,22 +22,22 @@ <h3 class="commercial__title">
             </div>
             <div class="commercial__body">
                 <ul class="lineitems">
-                    @events.take(4).map { event =>
+                    @masterclasses.take(4).map { masterclass =>
```

Comment:
Probably better to `take 4` in the controller and just pass the list to the template.